apidoc/v2/openapi.yaml

```diff
@@ -2265,7 +2265,7 @@ paths:
     post:
       tags: ['Experiments categories']
       operationId: post-team-one-expcat
-      summary: Create a new experiments status.
+      summary: Create a new experiments categories.
```

Comment:
Can you please check the lines down to 2367 and replace all "status" with "categories". Except "statuslike".

Reply:
It should be singular: Create a new category for experiments.

Reply:
Done.

Reply:
> It should be singular: Create a new category for experiments.

My bad! Bad copy pasta on my part.